### PR TITLE
Routing

### PR DESCRIPTION
**Pull request by ctlong**

### CODE REVIEW

**Comment by ctlong** [comment]
This is a quick one I promise, I already tested it in postman.

File: app/controllers/submissions_controller.rb [hunk 1]
```diff
@@ -23,4 +23,15 @@ def destroy
       end
     end
   end
+
+  def list
+    forms = User.find(params[:user_id]).forms
+    @submissions = []
```
Comment (h8rry) [inline app/controllers/submissions_controller.rb]: You can do `User.find(params[:user_id]).forms.joins(:submissions)`

Comment (h8rry) [inline app/controllers/submissions_controller.rb]: A join query is more effecient

**Comment by ctlong** [comment]
@ggharry I have reviewed and updated this pr

**Pull request merged**